sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/serializer/DigitalTwinsStringSerializer.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core.implementation.serializer;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import java.io.IOException;
+
+/**
+ * Configures the {@link StdSerializer<String>} with custom behavior needed to work with Digital Twins REST API.
+ */
+public class DigitalTwinsStringSerializer extends StdSerializer<String> {
+    private static final long serialVersionUID = 1L;
+    private final ObjectMapper mapper;
+
+    public DigitalTwinsStringSerializer(Class<?> t, boolean dummy, ObjectMapper mapper) {
```

Comment:
I think we need to rename this variable ... fake, mock or something

Reply:
I'll make use of the the following base class constructor instead:
```java
protected StdSerializer(Class<T> t) {
    this._handledType = t;
}
```

Weirdly enough, this is what the `StdSerializer` defines it as, which is why this signature was autogenerated:
```java
protected StdSerializer(Class<?> t, boolean dummy) {
    this._handledType = t;
}
```